src/backend/binaryen/index.ts

```diff
@@ -660,12 +657,8 @@ export class WASMGen extends Ts2wasmBackend {
             }
         }
         this.parseBody(func.body);
-        if (func.envParamLen > 0) {
-            this.currentFuncCtx.localVarIdxNameMap.set('context', 0);
-            if (func.envParamLen === 2) {
-                this.currentFuncCtx.localVarIdxNameMap.set('this', 1);
-            }
-        }
+        this.currentFuncCtx.localVarIdxNameMap.set('@context', 0);
```

Comment:
I remember we use `@` prefix in parameter, do we need to use this prefix also for variable?

Reply:
For `context`, we'd better add the prefix to distinguish it from ordinary variable, but for `this`, we should not add the prefix, since `this` may be used in body.
And I think it should follow to frontend, in frontend, we set variable name as `@context` and `this`:
https://github.com/intel/Wasmnizer-ts/blob/9229ddb87b9f3526083e4e9cc2b9de1ce0f2038a/src/scope.ts#L565
https://github.com/intel/Wasmnizer-ts/blob/9229ddb87b9f3526083e4e9cc2b9de1ce0f2038a/src/scope.ts#L856

Reply:
OK